homeassistant/components/light/zwave.py

```diff
@@ -244,9 +235,9 @@ def _value_added(self, value):
         # Check for the missing color values
         self._get_color_values()
 
-    def update_properties(self):
+    def update(self):
```

Comment:
I think that's a good enough reason to keep it as it was. I'll revert the rename.